wadium/story/tests/test_edit_story.py

```diff
@@ -0,0 +1,182 @@
+from django.test import TransactionTestCase, Client
+from rest_framework.authtoken.models import Token
+from rest_framework import status
+import json
+import datetime
+
+from story.models import Story
+from django.contrib.auth.models import User
+from .constants import body_example
+
+class PostStoryTestCase(TransactionTestCase):
+    client = Client()
+    reset_sequences = True
```

Comment:
[공식 문서](https://docs.djangoproject.com/en/3.1/topics/testing/advanced/)에 언급된대로, 
> Unless you are explicitly testing primary keys sequence numbers, it is recommended that you do not hard code primary key values in tests.
> 
> Using reset_sequences = True will slow down the test, since the primary key reset is a relatively expensive database operation.


reset sequence보다는 setUp()에서 story 생성 후 id를 받아서 그 뒤 테스트에 활용하는 방법이 좋은 것 같습니다.